__tests__/account/AccountCreateTransaction.test.ts

```diff
@@ -19,51 +19,75 @@ describe('AccountCreateTransaction', () => {
         const bodybytes = "Cg4KCAjcyQcQ258JEgIYAxICGAMYgMLXLyICCHhaPwoiEiDgyOwnWKWHn/rCJqE8DFFreZ5y41FBoN2Cj5TTeYiktzD//////////384//////////9/SgUI0MjhAw==";
 
         const tx = transaction.toProto().toObject();
-        expect(tx).toStrictEqual({
-            body: undefined,
+        let expectedTx = {
+            body: undefined as undefined | Object,
             bodybytes,
             sigmap: {
                 sigpairList: [
                     {
+                        pubkeyprefix: "4MjsJ1ilh5/6wiahPAxRa3mecuNRQaDdgo+U03mIpLc=",
                         contract: "",
-                        ecdsa384: "",
                         ed25519: "Vueml5zTmYI2f9O2kcw6/zKFaIPn5WCOKD/jvhwO+EFN55yepuYa566qbD8Z274nncCi/aqVrr/M6NIu91OnAQoOCggI3MkHENufCRICGAMSAhgDGIDC1y8iAgh4Wj8KIhIg4MjsJ1ilh5/6wiahPAxRa3mecuNRQaDdgo+U03mIpLcw//////////9/OP//////////f0oFCNDI4QM=",
-                        pubkeyprefix: "4MjsJ1ilh5/6wiahPAxRa3mecuNRQaDdgo+U03mIpLc=",
-                        rsa3072: ""
+                        rsa3072: "",
+                        ecdsa384: ""
```

Comment:
it's not enforced but these keys should probably be sorted for sanity reasons

Reply:
The reason these fields are sorted the way they are is because the stringified json won't match otherwise.

Reply:
Oh that's gross. At least it matches the ordering in the protobuf so it's hopefully stable.